firefox-ios/Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift

```diff
@@ -361,6 +380,8 @@ class HistoryPanelViewModel: FeatureFlaggable {
 
     private func getDeletableSection(for dateOption: HistoryDeletionUtilityDateOptions) -> [Sections]? {
         switch dateOption {
+        case .lastHour:
+            return [.today, .yesterday]
```

Comment:
what's this doing exactly, if we're doing .lastHour? Why is yesterday there?

Reply:
Currently, in the history tab, items are split into today, yesterday and and all time. I believe that when deleting last hour history, some of the history items may also end up in yesterday's section and as a result we should include it as part of the delete-able sections

Reply:
Currently in the history tab items are split into today, yesterday, last week, last month and older. I believe that when deleting last hour history, some of the history items may also end up in yesterday's section (ex: I search something at 11:50pm and try to clear history 15 mins later) and as a result we should include it as part of the delete-able sections.